### PR TITLE
DRACO PNTS support

### PR DESCRIPTION
**Pull request by Gmadges**
- Add correct code for parsing DRACO encoded PNTs
- hacky way of getting the DracoLoader

### CODE REVIEW

File: src/three/PNTSLoader.js [hunk 1]
```diff
@@ -1,53 +1,84 @@
...
 export class PNTSLoader extends PNTSLoaderBase {
 
 	constructor( manager = DefaultLoadingManager ) {
 
 		super();
 		this.manager = manager;
 
+		// very hacky way of getting the draco loader
+		this.dracoLoader = this.manager.getHandler( 'test.drc' );
+
```
Comment (Rennzie) [inline src/three/PNTSLoader.js]: Should we use something more descriptive than `test.drc`? Maybe `draco.drc`?

**Review by Rennzie — APPROVED** [review]

File: src/three/PNTSLoader.js (hunk 1 above)
Comment (Gmadges) [reply src/three/PNTSLoader.js]: sure

**Review by theoistrate — APPROVED** [review]

**Review by justinmanley — APPROVED** [review]
is this something we could contribute back upstream?

**Comment by Gmadges** [comment]
> is this something we could contribute back upstream?

Yep hopefully

**Pull request merged**